package.json

```diff
@@ -33,10 +33,10 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
+    "@angular/cli": "^1.0.0",
     "@angular/compiler-cli": "2.4.0",
```

Comment:
@13nehu I know about it. But, at present don't conclude these files here. They will give a build error.